Controller/DependentFilteredEntityController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Grossum\ExtendedFormTypeBundle\Controller;
+
+use Grossum\ExtendedFormTypeBundle\Services\DependentEntityLoader;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Security\Core\Authorization\Voter\AuthenticatedVoter;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+/**
+ * Class DependentFilteredEntityController
+ * @package Grossum\ExtendedFormTypeBundle\Controller
+ */
+class DependentFilteredEntityController extends Controller
+{
+    public function getOptionsAction(Request $request)
+    {
+        $entities = $this->container->getParameter('grossum.dependent_filtered_entities');
+        $entityInf = $entities[$request->get('entity_alias')];
```

Comment:
What happens when value of `entity_alias` is not contained in `$entities` array?